### ISSUE TITLE
Input and output sizes of images？

### ISSUE THREAD

**Issue by ArtScanner** [phillipi/pix2pix#195]
Why do I find that the output size will be smaller in the training results？
How can I modify the code to customize the output size or the same size as the input image...

**Comment by phillipi** [comment]
`fineSize` specifies the output image size, e.g., `DATA_ROOT=./datasets/facades name=facades_generation which_direction=BtoA fineSize=256 th train.lua`

**Comment by MoeinSorkhei** [comment]
Hi @phillipi,

I have a question about the Cityscapes output images in your paper. You mentioned that you evaluate the model on 256x256 images, while the visual examples that you show in your paper are rectangular rather than square. 

Do you manually resize your 256x256 images into rectangular images in your paper or do you train another separate model on rectangular images?

Thanks in advance.

**Comment by phillipi** [comment]
For visualization we resized to rectangular images, but the model always output 256x256.

**Comment by MoeinSorkhei** [comment]
Hi @phillipi ,
Thanks for the response.
I wanted to ask the same question about the **photo->label** direction. 
Do you also resize the images to 256x256 for training, and then back to the original size1024x20148 for evaluation of the generated segmentations? (I could not find this detail in your paper)

Thanks in advance.

**Comment by phillipi** [comment]
That's right, it is trained at 256x256. For evaluation, more information is here: [1](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/qa.md#for-labels2photo-cityscapes-evaluation-why-does-the-pretrained-fcn-8s-model-not-work-well-on-the-original-cityscapes-input-images-150), [2](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/tips.md#evaluating-labels2photos-on-cityscapes)

My advice is one should almost never compare numbers _between_ papers, only _within_ papers. So don't worry if your eval protocol isn't exactly the same as what we did, as long as yours is reasonable. If you have a new method you are trying, then run your evaluation code on both your new method and on the baseline to directly compare the results.

**Comment by MoeinSorkhei** [comment]
Great! Thanks for your feedback!
